src/Employer/Employer.Web/ViewModels/VacancyPreview/VacancyPreviewViewModel.cs

```diff
@@ -444,7 +448,17 @@ public enum VacancyTaskListSectionState
         InProgress,
         Completed
     }
-    
-    
+
+    public class MustBeTrueAttribute : ValidationAttribute
```

Comment:
We don't need this attribute - if we did then it should go into the shared part as its used by employer and provider, and not nested in this view model.

Reply:
Removed.